snekcord/clients/wsevents.py

```diff
@@ -201,7 +201,31 @@ async def execute(cls, client, shard, payload):
                 emojis.add(guild.emojis.upsert(emoji).id)
 
             for emoji_id in set(guild.emojis.keys()) - emojis:
-                del guild.emojis.mapping[emoji_id]
+                guild.emojis[emoji_id]._delete()
+
+        return cls(shard=shard, payload=payload, guild=guild)
+
+    @property
+    def partial(self):
+        return self.guild is None
+
+
+@register('GUILD_STICKERS_UPDATE', intent='GUILD_EMOJIS_AND_STICKERS')
+class GuildStickersUpdate(BaseEvent):
+    _fields_ = ('guild',)
+
+    @classmethod
+    async def execute(cls, client, shard, payload):
+        guild = client.guilds.get(Snowflake(payload['guild_id']))
+
+        if guild is not None:
+            stickers = set()
+
+            for sticker in payload['stickers']:
+                stickers.add(guild.stickers.upsert(sticker).id)
+
+            for sticker_id in set(guild.stickers.keys()) - stickers:
```

Comment:
```suggestion
            for sticker_id in guild.stickers.keys() - stickers:
```

Reply:
I don't want to change this yet, I might change `keys()` to return something else in the future

Reply:
oh ok